src/desktop/components/artwork_metadata_stub/templates/contact.jade

```diff
@@ -22,7 +22,7 @@ if artwork.sale && artwork.sale.is_auction
           | #{sa.opening_bid.display}
       else if artwork.sale.is_closed
         | Auction closed
-else if artwork.is_inquireable && !(artwork.is_acquireable && auctionPartner)
+else if artwork.is_inquireable && !(artwork.is_acquireable && (auctionPartner || sd.ENABLE_NEW_BUY_NOW_FLOW))
```

Comment:
It feels kind of weird that these acquirable works don't display 'Buy' in this rail?

(I realize this might be out of the scope for now, but we support that in _yet another_ artwork brick/tombstone implementation- https://github.com/artsy/force/blob/72b9f62f225bca6a1601e572e9137a91a3bf5901/src/desktop/components/artwork_item/templates/artwork.jade#L29-L34 . There, we render 'buy' if acquirable, otherwise the contact line, if inquireable. That feels better than the conditional here).

Reply:
Yeah, seems out of scope for this bugfix. Could this just be replaced by the new brick?